How-to-push-to-nuget.txt

```diff
@@ -1,4 +1,7 @@
-﻿ - Update the version number in the project file
+﻿AppVeyor is configured to deploy to NuGet when master is pushed,
+so following these steps steps should not be necessary in general.

```

Comment:
Is dit verstandig?

Kunnen we nog traceren welke versie de code is zonder AppVeyor?

Reply:
Volgens mij gaat eamons vraag eigenlijk over regel 1 in appveyor.yml.  Ik weet niet wat de regels zijn, maar we zouden kunnen zeggen dat het versienummer de sha bevat?

Reply:
> Kunnen we nog traceren welke versie de code is zonder AppVeyor?

Ik denk het niet en dat konden we in de oude situatie ook niet goed. SymbolSource zou wel een versie van de package met de source moeten hebben.

> maar we zouden kunnen zeggen dat het versienummer de sha bevat?

Niet als het geen preview-versie is geloof ik.

Reply:
In de oude versie stond het versie-nummer in de source code.  Je weet niet welke versie daadwerkelijk gepushed is, nee.

Reply:
Dus het gaf hooguit een indicatie.

We zouden de patching uit kunnen zetten, maar ik denk dat het nu beter is eigenlijk.

Reply:
Ik vind het probleem twijfelachtig. Volgens mij zijn alle minor releases goed te traceren.

Reply:
Dit pushed alles als patch-level wijziging, ook al is het helemaal geen kleine wijziging, dus dat is niet wat je als consument zou verwachten.  En het is helemaal niet te traceren vanuit de repo; we introduceren dus een afhankelijkheid op nog een service.